content/en/docs/instrumentation/js/getting-started/nodejs.md

```diff
@@ -88,11 +138,30 @@ npm install @opentelemetry/auto-instrumentations-node
 
 The tracing setup and configuration should be run before your application code. One tool commonly used for this task is the [`-r, --require module`](https://nodejs.org/api/cli.html#cli_r_require_module) flag.
 
-Create a file with a name like `tracing.js` which will contain your tracing setup code.
+Create a file named `tracing.ts|js`, which will contain your tracing setup code.
 
-```javascript
-/* tracing.js */
+{{< ot-tabs TypeScript JavaScript >}}
 
+{{< ot-tab >}}
+/*tracing.ts*/
+// Require dependencies
+import * as opentelemetry from "@opentelemetry/sdk-node";
+import { getNodeAutoInstrumentations } from "@opentelemetry/auto-instrumentations-node";
+import { diag, DiagConsoleLogger, DiagLogLevel } from '@opentelemetry/api';
+
+// For troubleshooting, set the log level to DiagLogLevel.DEBUG
+diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.INFO);
+
+const sdk: any = new opentelemetry.NodeSDK({
```

Comment:
If the explicit typing is `any`, I'd suggest leaving it for TypeScript to deduce the correct type.

```suggestion
const sdk = new opentelemetry.NodeSDK({
```

Reply:
Oh, that makes sense. I have removed it @legendecas